testsuite/features/secondary/srv_health_check_supportconfig.feature

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2025 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@scope_salt
+Feature: Health Check tool based on a supportconfig
+  A supportconfig generated from the Uyuni server
+  should be able to be parsed by Health Check tool.
+
+  Scenario: A supportconfig is taken from the server
+    When I generate a supportconfig for the server
+    Then I obtain and extract the supportconfig from the server
+
+  Scenario: Install health check tool
+    When I setup a health check environment on the controller
+    Then file "/usr/lib/python3.11/site-packages/health_check/config.toml" should exist on "localhost"
+    And I run "mgr-health-check --help" on "localhost"
```

Comment:
Please, pre-install this tool on the controller through sumaform.
It's good to decouple the provision of our test environment from the system's tests.